nodemon.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
FYI: Sails has some efforts in terms of "watch" functionality: https://github.com/balderdashy/sails/issues/366